spanish_(spain)/global.ini

```diff
@@ -403,84 +403,84 @@ Civilian_RepUI_Founded,P=[PH] N/A
 Civilian_RepUI_Headquarters,P=[PH] Cuartel general civil
 Civilian_RepUI_Leadership,P=[PH] Liderazgo civil
 Civilian_RepUI_Name,P=[PH] Civil
-Clovis_Safe_WIP_Journal_Sub_Heading_Clue_01=Incapaz de abrir a puerta segura por favor envie el codigo
-Clovis_Safe_WIP_Journal_Sub_Heading_Clue_02=El codigo para abrir la caja fuerte es 459256
-Clovis_Safe_WIP_Journal_Title_Clue_01=Incapaz de abrir seguro
-Clovis_Safe_WIP_Journal_Title_Clue_02=RE: Incapaz de abrir seguro
-Clovis_Safe_WIP_Mission_Title_Text=Investigar seguro
-Clovis_Safe_WIP_Objective-001_Text_Long=Ir a la ubicacion de la caja fuerte que falta
-Clovis_Safe_WIP_Objective-001a_Text_Long=Encontrar y recoger la caja fuerte
-Clovis_Safe_WIP_Objective-002_Text_Long=Devolver la caja fuerte al administrador
-Clovis_Safe_WIP_Objective_001_Text_Short=Ir a ubicacion segura
-Clovis_Safe_WIP_Objective_001a_Text_Short=Encontrar seguro
-Clovis_Safe_WIP_Objective_002_Text_Short=Volver seguro
-Clovis_Safe_WIP_Objective_003_Text_Long=Investiga las pistas que podrian proporcionar un codigo para desbloquear la caja fuerte.
-Clovis_Safe_WIP_Objective_003_Text_Short=Investigar pistas
-Clovis_Safe_WIP_Objective_003a_Text_Long=Encuentra el segundo datapad para el codigo de la caja fuerte.
-Clovis_Safe_WIP_Objective_003a_Text_Short=Encuentra el siguiente datapad
-Clovis_Safe_WIP_Objective_004_Text_Long=Regresa a la caja fuerte y desbloquea el cofre.
-Clovis_Safe_WIP_Objective_004_Text_Short=volver a salvo
-Clovis_Safe_WIP_Objective_005_Text_Long=Devuelve el objeto de la mision al administrador.
-Clovis_Safe_WIP_Objective_005_Text_Short=Articulo de mision de retorno
-ClovusDarneely_RepUI_Association=Recuperacion y eliminacion
-ClovusDarneely_RepUI_Biography=Clovus Darneely es empleado de Reclamation & Disposal, el principal deposito de salvamento de Lorville. Con una reputacion de ser muy educado y un poco erudito, no esta claro entre sus clientes por que trabaja en un puesto tan remedial.
-ClovusDarneely_RepUI_Location=Lorville, Hurston, sistema Stanton
-ClovusDarneely_RepUI_Occupation=Empleado
+Clovis_Safe_WIP_Journal_Sub_Heading_Clue_01=No se puede abrir la puerta de la caja fuerte, por favor, envien el codigo.
+Clovis_Safe_WIP_Journal_Sub_Heading_Clue_02=-FALLO AL ENVIAR EL MENSAJE-\nEl codigo para abrir la caja fuerte es 459256.
+Clovis_Safe_WIP_Journal_Title_Clue_01=Imposible Abrir la Caja Fuerte
+Clovis_Safe_WIP_Journal_Title_Clue_02=RE: Imposible Abrir la Caja Fuerte
+Clovis_Safe_WIP_Mission_Title_Text=Investigar la Caja Fuerte
+Clovis_Safe_WIP_Objective-001_Text_Long=Ir al lugar de la caja fuerte desaparecida.
+Clovis_Safe_WIP_Objective-001a_Text_Long=Encontrar y recoger la caja fuerte.
+Clovis_Safe_WIP_Objective-002_Text_Long=Devolver la caja fuerte al administrador.
+Clovis_Safe_WIP_Objective-001_Text_Short=Ir a la ubicacion de la caja fuerte.
+Clovis_Safe_WIP_Objective-001a_Text_Short=Encontrar la caja fuerte.
```

Comment:
Hay algún motivo por el que reemplazas la variable? De Clovis_Safe_WIP_Objective_001_Text_Short a Clovis_Safe_WIP_Objective-001_Text_Short (se diferencia en el guión entre objective y 001

Reply:
pues no ha sido cambiado intencionadamente.
Pero he revisado el global.ini original en ingles que tenéis subido en https://raw.githubusercontent.com/Autovot/StarCitizen-Localization/main/data/Localization/english/global.ini
que es el que uso de referencia para ver la versión original, y ahí viene con guion medio "-"
![image](https://github.com/Autovot/SC_Spanish_SOK/assets/12573214/0ba55c05-ae52-4dfa-a813-5dd89481724b)
de ahí que al llevarlo a la rama se haya cambiado. Ahora desconozco cual de las 2 versiones es la correcta. Entiendo que será la del guion bajo "_" por que todas las claves vienen así.